### DISCUSSION TITLE
Grid for plots

### DISCUSSION THREAD

**Discussion by daniel-tucano** [JetBrains/lets-plot-kotlin#88] [Ideas]
I think grids are a basic feature of the plots that are missing in the library. It would be nice to have those :)

**Comment by alshan** [comment]
Agree ) Also trying to figure out what other cosmetic improvements could work best.

**Comment by alshan** [comment]
Hi @daniel-tucano , we've added better themes support in v3.1.0 which includes gridlines as well. Take a look: https://github.com/JetBrains/lets-plot-kotlin#what-is-new-in-310
